cfgmgr/intfmgr.cpp

```diff
@@ -447,9 +447,20 @@ std::string IntfMgr::setHostSubIntfMtu(const string &alias, const string &mtu, c
     }
     SWSS_LOG_INFO("subintf %s active mtu: %s", alias.c_str(), subifMtu.c_str());
     cmd << IP_CMD " link set " << shellquote(alias) << " mtu " << shellquote(subifMtu);
-    EXEC_WITH_ERROR_THROW(cmd.str(), res);
+    std::string cmd_str = cmd.str();
+    int ret = swss::exec(cmd_str, res);
 
-    return subifMtu;
+    if (ret && !isIntfStateOk(alias))
```

Comment:
This is already checked in the `doIntfGeneralTask `under SET operation. So why check again here? is there a race condition?

Reply:
We are not visiting the `"doIntfGeneralTask"` function in this flow.
The flow is:
1. changing the "mtu" field in the PORT_TABLE in state DB
2. `doTask`
3. `doPortTableTask`
4. `updateSubIntfMtu`
5. `setHostSubIntfMtu`

Reply:
ok, does this happen in the `updateSubIntfAdminStatus `flow as well? Or why only `mtu `change is handled?